packages/devtools/lib/src/config_specific/allowed_error.dart

```diff
@@ -0,0 +1,6 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+export 'allowed_error_default.dart'
+    if (dart.library.html) 'allowed_error_html.dart';
```

Comment:
Config specific exports don't work everywhere yet. We may need to be careful how we use this. They don't work internally for instance.

Reply:
Any eta on when they will work internally?
Fyi @DaveShuckerow as this could impact what you are working on.

Reply:
cc @jakemac53 

I'm not sure the plans around this.
I think that some of the discussions that @kevmoo started around making platform specific imports more limited may have also involved removing support for conditional exports... I could be misremembering though.

Reply:
Once we remove DDC internally then they should ~work. The analyzer is only ever going to look at a single one of the imports though (depending on the analyzer summary that is used).

Reply:
I kept this using config specific imports. If we hit issues downstream we can swap this out for some other technique.